sklearn/calibration.py

```diff
@@ -1295,8 +1295,15 @@ def from_estimator(
         if not is_classifier(estimator):
             raise ValueError("'estimator' should be a fitted classifier.")
 
-        y_prob, pos_label = _get_response(
-            X, estimator, response_method="predict_proba", pos_label=pos_label
+        check_is_fitted(estimator)
+        if len(estimator.classes_) != 2:
+            raise ValueError("Estimator must be a binary classifier.")
+
+        y_prob, pos_label = _get_response_values(
+            estimator,
+            X,
+            response_method="predict_proba",
+            pos_label=pos_label,
         )
```

Comment:
We could use _get_response_binary here

Reply:
OK so it makes sense to move `_get_response_binary` then.